module_utils/foreman_helper.py

```diff
@@ -244,6 +256,9 @@ def _generate_resource_params(self, resource, action, resource_id=None, params=N
                 resource_params[hash_entry_name] = params
             else:
                 resource_params.update(params)
+        # In combination with katello, the organization entity needs its name provided also as a plain parameter
```

Comment:
Could you please reference https://projects.theforeman.org/issues/26855 here

Reply:
or, could you try https://github.com/theforeman/foreman-ansible-modules/pull/276 which should be the cleaner hack for this?

Reply:
or we have a look at https://github.com/theforeman/hammer-cli/blob/16a30c6e93cecbbf5321f402b1abb75ca428a8b4/lib/hammer_cli/apipie/options.rb#L9-L27 and see how hammer fills out the params, at it works fine there.

Reply:
yeah, that's what I did now, and it works!

https://github.com/theforeman/foreman-ansible-modules/pull/277